src/sentry/api/serializers/models/exporteddata.py

```diff
@@ -8,8 +8,10 @@
 class ExportedDataSerializer(Serializer):
     def get_attrs(self, item_list, user, **kwargs):
         attrs = {}
+        users = User.objects.filter(id__in=set([item.user_id for item in item_list]))
+        user_lookup = {user.id: user for user in users}
         for item in item_list:
-            user = User.objects.get(id=item.user_id)
+            user = user_lookup[item.user_id]
             serialized_user = serialize(user)
```

Comment:
@wedamija suggested this type of implementation in the previous PR, to reduce querying